File: find_N_unique_integers_sum_up_to_zero.py
class Solution:
    def sumZero(self, n: int) -> int:
        result = []
        
        if n % 2 != 0:
            result.append[0]
            
        new_nums = n // 2
        
        for i in range(1, new_nums + 1, 1):
           result.append(i) 
           result.append(-i)
        
        return result 
        

File: findNUniqueIntegersSumUpToZero.js
/**
 * @param {number} n
 * @return {number[]}
 */
var sumZero = function(n) {
    let result = []
    if(n % 2 !== 0){
        return result.append(0)
    }

    for(let i = 0; i <= n / 2; i++){
        result.append(i, -i)
    }
    return result

}